### PR TITLE
Change map size from backbone router

### PR DESCRIPTION
**Pull request by dimasciput**
This will change the map size based on the url.

### CODE REVIEW

**Comment by cchristelis** [comment]
Let's merge

File: django_project/feti/static/feti/js/feti-backbone/feti-route.js [hunk 1]
```diff
@@ -6,19 +6,20 @@ var AppRouter = Backbone.Router.extend({
     routes: {
         "": "landing_page",
         "/login": "login_page",
-        "/map/:mode": "show_map",
+        "map/:mode": "show_map",
```
Comment (cchristelis) [inline django_project/feti/static/feti/js/feti-backbone/feti-route.js:9]: Hi @dimasciput, why did you remove the leading slash here?

Comment (dimasciput) [reply django_project/feti/static/feti/js/feti-backbone/feti-route.js:9]: @cchristelis it doesn't work with leading slash. I think I should remove the others.
The router example : http://backbonejs.org/#Router
